src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -57,16 +57,43 @@
     private static final String TAG_WORK                       = "work";
     private static final String TAG_MANUAL_HIRING              = "manualHiring";
     private static final String TAG_WAYPOINT                   = "waypoints";
-    private static final String TAG_BLOCK                      = "blockState";
+
+    //statistics tags
+    private static final String TAG_STATISTICS                 = "statistics";
+    private static final String TAG_MINER_STATISTICS           = "minerStatistics";
+    private static final String TAG_MINER_ORES                 = "ores";
+    private              int    minedOres                      = 0;
+    private static final String TAG_MINER_DIAMONDS             = "diamonds";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "diamonds" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1192)